yarn-error.log

```diff
@@ -0,0 +1,9392 @@
+Arguments: 
```

Comment:
@jalextowle can you add this to the `.gitignore`?

Reply:
Hmm, I already put `*/**/yarn-error.log` in the .gitignore 😕